FirebaseCore/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   option `use_frameworks! :linkage => :static` to get the Firebase 6.x linkage behavior. (#2022)
 - [removed] Removed broken `FirebaseOptions()` initializer. Use `init(contentsOfFile:)` or
   `init(googleAppID:gcmSenderID:)` instead. (#6633)
+- [changed] All Firebase pods now have the same version.
+- [changed] In CocoaPods, Firebase betas are now indicated in the version tag. In SwiftPM, beta
+  is appended to the product name.
+- [changed] The version must now be specified for the two Swift-only Firebase CocoaPods in the
+  Podfile like `pod 'FirebaseFirestoreSwift', '~> 7.0-beta'`.
```

Comment:
This should this include the patch version, right?

Reply:
Not necessarily. It's standard semantic versioning. '~> 7.0-beta'` allows minor version updates. '~> 7.0.0-beta'` will allow only patch updates.

Reply:
Gotcha, thanks. I wasn't sure if it would pick it up appropriately given it's not a simple `X.Y.Z` string.